.web-ext-config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	sourceDir: 'extension',
+	run: {
+		keepProfileChanges: true,
```

Comment:
This warning from the docs should be repeated in the guidance if a non-temporary profile is recommended:

> This option makes the profile specified by --firefox-profile completely insecure for daily use. It turns off auto-updates and allows silent remote connections, among other things. Specifically, it will make destructive changes to the profile that are required for web-ext to operate.

Reply:
Done